metaflow/plugins/kfp/tests/.gitlab-ci.yml

```diff
@@ -163,11 +162,10 @@ test:nonprod:
         -e AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION 
         ${BUILT_IMAGE_FULL_PATH}
         bash -c "
-          export KFP_RUN_URL_PREFIX=https://kubeflow.corp.stage-k8s.zg-aip.net/ && 
-          export KFP_SDK_NAMESPACE=metaflow-integration-testing-dev && 
-          export METAFLOW_DATASTORE_SYSROOT_S3=s3://aip-example-stage/metaflow/ && 
-          export METAFLOW_USER=${GITLAB_USER_EMAIL} &&
-          cd /home/zservice/metaflow/metaflow/plugins/kfp/tests && 
+          export KFP_RUN_URL_PREFIX=https://kubeflow.corp.stage-k8s.zg-aip.net/ &&
+          export KFP_SDK_NAMESPACE=metaflow-integration-testing-dev &&
+          export USER=${GITLAB_USER_EMAIL} &&
```

Comment:
I recognize the where the datastore change come from, but I am not quite sure what caused changes from `METAFLOW_USER` to `USER`. Are they both from your changes in kubeflow namespace (profile)?

Reply:
Thanks for asking.  If it is METAFLOW_USER then the run gets tagged as that user, in this case the GITLAB_USER_EMAIL.  see [code](https://github.com/zillow/metaflow/blob/88a639649d818c194d26824e706641cde63bf7f6/metaflow/plugins/kfp/kfp.py#L716-L720).

It's cleaner if the run is a `kfp-user` (an automated run 

USER doesn't persist and would then only be used for creating the pipeline see [code](https://github.com/zillow/metaflow/blob/88a639649d818c194d26824e706641cde63bf7f6/metaflow/plugins/kfp/kfp.py#L157-L165)